test/e2e/kubernetes/pod/pod.go

```diff
@@ -818,8 +820,20 @@ func (p *Pod) CheckWindowsOutboundConnection(url string, sleep, duration time.Du
 						readyCh <- false
 					}
 				} else {
-					log.Printf("Error:%s\n", err)
-					log.Printf("Out:%s\n", out)
+					// in case bing.com is down let's hope google.com is also not down
```

Comment:
I would remove this comment.

or make it something like "try with a second URL in the case the first one is down"